com.woltlab.wcf/templates/trophyList.tpl

```diff
@@ -3,7 +3,7 @@
 {capture assign='contentHeader'}
 	<header class="contentHeader messageGroupContentHeader">
 		<div class="contentHeaderTitle">
-			<h1 class="contentTitle">{$category->getTitle()}</h1>
+			<h1 class="contentTitle">{$category->getTitle()} <span class="badge badgeInverse">{#$items}</span></h1>
```

Comment:
I know, and I fixed it here... I'm really sure... And I can't fix that somehow, because at my local version, this already has been fixed and git won't commit that?